src/components/apiDelegation/reusables/OverviewPageContent/OverviewPageContent.module.css

```diff
@@ -43,4 +43,16 @@
 .pageContentText {
   font-weight: normal;
   margin-top: -2rem;
-}
\ No newline at end of file
+}
+
+.infoPanelWrapper{
```

Comment:
According to the mobile design for this page, the panel should span the entire page horisontally.
If this is feasible, I think it would look neat and also give us more space for the problematic header.
![image](https://user-images.githubusercontent.com/22012997/216377028-9e76b9a2-ce1b-460b-a25d-d57d1a48436c.png)